### PR TITLE
Issue #542 - Browse Page: revert function doesn't work as expected

### LINKED ISSUE #542: Browse Page: revert function doesn't work as expected
While applying some changes to the definitions in the Browse page, clicking on the Revert Button of a single Definition, or using the Revert Changes at the top of the list, actually doesn't revert anything.

### PR DESCRIPTION
**Pull request by storrisi**
Updated revert method to work with both definition and browse store

Fixes #542

### CODE REVIEW

**Comment by storrisi** [comment]
You can now review this PR

**Comment by jeffmcaffer** [comment]
Love the inclusion of way more tests. THANKS!

File: src/components/SystemManagedList.js [hunk 1]
```diff
@@ -286,13 +291,15 @@ export default class SystemManagedList extends Component {
...
+            dispatch(uiRevert(definition, null, store))
             this.incrementSequence()
             notification.close(key)
           }}
           onClose={() => notification.close(key)}
-          confirmText="OK"
-          dismissText="Cancel"
+          confirmButtonTestId="notification-revert-confirm"
+          dismissButtonTestId="notification-revert-dismiss"
+          confirmText="Revert"
```
Comment (jeffmcaffer) [inline src/components/SystemManagedList.js]: Please stick with OK and Cancel throughout the site.

**Comment by storrisi** [comment]
@jeffmcaffer i've updated the labels.

**Pull request merged**